app/javascript/angular/code/controllers/_fixed_sessions_map_ctrl.js

```diff
@@ -129,10 +137,38 @@ export const FixedSessionsMapCtrl = (
         params.update({data: {usernames: profiles.join(", ")}});
         $scope.sessions.fetch();
       });
+
+      setTimeRangeFilter(params, elmApp)
+
+      elmApp.ports.updateTimeRange.subscribe((timeRange) => {
+        params.update({data: {timeFrom: timeRange.timeFrom, timeTo: timeRange.timeTo}});
+        $scope.sessions.fetch();
+      });
     });
   }
 }
 
+const setTimeRangeFilter = (params, elmApp) => {
+  if (document.getElementById("daterange")) {
+    $('input[id="daterange"]').daterangepicker({
+      opens: 'left',
+      timePicker: true,
+      startDate: moment.unix((params.get('data').timeFrom)).utc().format('DD/MM/YYYY hh:mm A'),
+      endDate: moment.unix((params.get('data').timeTo)).utc().format('DD/MM/YYYY hh:mm A'),
```

Comment:
May be better to pass `timeFrom` and `timeTo` instead of `params` to `setTimeRangeFilter`. The api of that module (ie `get(String).key`) looks ugly. If we keep it closer to Angular it's gonna be easier to get rid of